llvmlite/tests/test_binding.py

```diff
@@ -632,6 +632,7 @@ def test_emit_assembly(self):
         target_machine = self.target_machine()
         mod = self.module()
         ee = self.jit(mod, target_machine)
+        target_machine.set_verbose(True)
```

Comment:
I think we should test both the non-verbose and the verbose case (perhaps with different checks?).